utils/growpart/Makefile

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (C) 2022 Eero Vuojolahti
+#
+# This is free software, licensed under the GNU General Public License v2.
```

Comment:
please use spdx

Reply:
What is that? I copied the Makefile template from some other package and only changed the year and the name.